src/systems/ec/implementations/vanilla/packages/ec-component-list/ec-component-list.scss

```diff
@@ -0,0 +1,127 @@
+/**
+ * List component
+ */
+
+// Import base
+@import '@ecl/ec-base/ec-base';
+
+/**
+ * @define ordered-list
+ */
+@mixin ecl-ordered-list(
+  $text-color: $ecl-color-grey-100,
+  $divider-color: $ecl-color-grey-15
+) {
+  .ecl-ordered-list {
+    color: $text-color;
+    font: $ecl-font-prolonged-m;
+    margin: 0;
+    // Approximately centered bullet
+    padding-left: calc((#{$ecl-spacing-s} * 2) + 0.3em);
```

Comment:
you expected me to leave a comment here... didn't you? "centered" compared to what?
I see that the examples from the specs are too simple, what should the list look like when the counter reaches 100 for example?

Reply:
"centered" as in "the same space before and after the bullet" (spacing-s).
The approximate comes from the `0.3em`, which is the size of the bullet (more or less).
But I agree that it is relevent mostly for unordered list.